examples/axisartist/demo_floating_axis.py

```diff
@@ -1,30 +1,35 @@
 """
-An experimental support for curvilinear grid.
+Axis Within Rectangular Frame
```

Comment:
@NelleV Do we want to capitalize every word in the sphinx-gallery docstring title, or only the first word?  I've been doing only the first word like in the example [here](http://matplotlib.org/devdocs/devel/MEP/MEP12.html).  I know this seems nitpicky, but a style mismatch will stand out in the sphinx gallery view. :P